databay/link.py

```diff
@@ -107,13 +106,15 @@ def add_inlets(self, inlets: Union[Inlet, List[Inlet]]):
             inlets = [inlets]
 
         for inl in inlets:
-            assert isinstance(inl, Inlet)
-            
+            if not isinstance(inl, Inlet):
+                raise TypeError("inl is not an instance of Inlet()")
```

Comment:
Similar suggestion as previously, however here I'd also point out that our users have reason to know about our private variable names, therefore `inl` is not too clear.